src/AcceptanceTesting/SpecByExample/SFA.DAS.Payments.AcceptanceTests/DataCollectors/SubmissionDataLockResultCollector.cs

```diff
@@ -87,7 +87,7 @@ private static void CreateOrUpdateDataLockPeriodResults(List<SubmissionDataLockP
                         {
                             CommitmentId = entity.CommitmentId,
                             CommitmentVersion = entity.CommitmentVersion,
-                            TransactionType = (TransactionType) entity.TransactionType
+                            TransactionType= (TransactionType) entity.TransactionType
                         }
                     }
                 });
```

Comment:
In the CreateOrUpdateDataLockPeriodResults method, the periodREsults parameter could be an ICollection.
Also, collectionPeriodNumber is never used so could be removed.